docusaurus/docs/iOS/01-basics/02-installation.mdx

```diff
@@ -23,6 +23,20 @@ To add `StreamVideo` SDK, open Xcode and follow these steps:
 
 ![Screenshot shows selection of dependencies](../assets/spm_select.png)
 
+<details><summary>➤ Do you want to use <b>XCFrameworks</b>? (Click to read more)</summary>
+<p>
+
+:::caution
```

Comment:
This one isn't probably needed as the Xcframework section contains it

Reply:
We decided to leave it in two places same way as done on Chat.